docs/Config_Reference.md

```diff
@@ -2082,6 +2082,26 @@ printer kinematics.
 #   MANUAL_STEPPER movement commands.
 ```
 
+### [mixing_extruder]
+
+A mixing printhead which has <n>in-1out mixing nozzle. When activated
+additional g-code are available. See [G-Codes](G-Codes.md#mixing_extruder)
+for a detailed description of the additional commands.
+
+```
+[mixing_extruder]
```

Comment:
It seems odd that this doesn't have a name as in `[mixing_extruder myname]`.

Reply:
I don't see for what the name can be used.